bin/common/nss_wrapper.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2021 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CRUNCHY_DIR=${CRUNCHY_DIR:-'/opt/crunchy'}
+
+# The following sets up an nss_wrapper environment in accordance with OpenShift
+# guidance for supporting arbitrary user ID's
+
+# define nss_wrapper directory and passwd & group files that will be utilized by nss_wrapper
+NSS_WRAPPER_DIR="/tmp/nss_wrapper/${NSS_WRAPPER_SUBDIR}"
+NSS_WRAPPER_PASSWD="${NSS_WRAPPER_DIR}/passwd"
+NSS_WRAPPER_GROUP="${NSS_WRAPPER_DIR}/group"
```

Comment:
@andrewlecuyer shouldn't we just call `source nss_wrapper_env.sh` here?

Reply:
OK. May be good to add a comment to that effect so someone does not attempt to optimize it.

Reply:
Comment added.